packages/ui/src/components/timeline/Timeline.module.scss

```diff
@@ -283,6 +275,15 @@
   }
 }
 
+.rangeContainer {
```

Comment:
To remain consistent with other components, this should be called `rangeTrack`
```suggestion
.rangeTrack {
```
![image](https://github.com/motion-canvas/motion-canvas/assets/64662184/5776adf9-ad82-498c-8fd4-0fe05e8bcfae)